Tribler/Core/defaults.py

```diff
@@ -251,6 +252,7 @@
 tribler_defaults['Tribler']['window_height'] = 670
 tribler_defaults['Tribler']['sash_position'] = -185
 tribler_defaults['Tribler']['family_filter'] = 1
+tribler_defaults['Tribler']['monochrome_tray_icon'] = False
```

Comment:
I did the change. Maybe I should be checking inside the function whether tray exists? 

Some test is failing because of timeout. Not sure whether it's my false or the test can be restarted.